packages/svelteui-core/src/components/Chip/Chip.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import useStyles from './Chip.styles.js';
+	import { randomID } from '$lib/styles';
+	import { get_current_component } from 'svelte/internal';
+	import { createEventForwarder, useActions } from '$lib/internal';
+	import Box from '../Box/Box.svelte';
+	import type { ChipProps as $$ChipProps } from './Chip.styles.js';
+
+	interface $$Props extends $$ChipProps {}
+
+	export let use: $$Props['use'] = [],
+		element: $$Props['element'] = undefined,
+		className: $$Props['className'] = '',
+		override: $$Props['override'] = {},
+		color: $$Props['color'] = 'blue',
+		id: $$Props['id'] = randomID(),
+		disabled: $$Props['disabled'] = false,
+		value: $$Props['value'] = null,
+		checked: $$Props['checked'] = false,
+		label: $$Props['label'] = null,
+		radius: $$Props['radius'] = 'sm',
```

Comment:
Perhaps make the default radius `xl` and size `sm`, to be more like a chip default size by default